### ISSUE TITLE
Import Workflow not working on-prem with big workflows

### ISSUE THREAD

**Issue by Alb-310** [Shuffle/Shuffle#1016]
When I try to import a big workflow ( json with 220k characters ) from an on prem instance to another, the workflow is empty.

In the logs I'm getting this error:

```
2023/01/11 10:24:06 [WARNING] Failed saving workflow to database: Bad statuscode from database: 400. Reason: {
  "error" : {
    "root_cause" : [
      {
        "type" : "illegal_argument_exception",
        "reason" : "mapper [comments.position.x] cannot be changed from type [float] to [long]"
      }
    ],
    "type" : "illegal_argument_exception",
    "reason" : "mapper [comments.position.x] cannot be changed from type [float] to [long]"
  },
  "status" : 400
}
```

**Comment by Alb-310** [comment]
when all the float on position x and y are rounded to the nearest int, the workflow import worked perfectly

**Comment by frikky** [comment]
Hey! I think this should be solved in 1.1.0, as I know we've solved this a while back.

What version are you on?

**Comment by Alb-310** [comment]
1.1.0

**Comment by frikky** [comment]
> 1.1.0

Hmm, I wonder if this is indeed then happening due to rounding errors on initialization of the Workflow index in elasticsearch itself, the first time a comment is used. 

Indeed doing as you said should then work as a workaround: setting the positions to whole integers in the spec itself

I'm setting this as a backend bug to be fixed for 1.2.0. Thanks!

**Comment by 0x0elliot** [comment]
Fixed with 1.3.0
